### ISSUE TITLE
Some hinges no longer respond to sequencer blocks and blueprint image doesn't update.

### ISSUE THREAD

**Issue by CurioInventorium** [tsunamayo/Starship-EVO#3830]
### Short Description of the issue:
Some of the hinges used in parts of my ship either stop responding to sequencer blocks or only move small amount.
And the blueprint image doesn't seem to have updated for awhile.
...

### Steps to reproduce the issue:
I'm unsure exactly how to reproduce this. As when I first built the cargo door the primary hinge worked fine. At some point it would no longer move to the angles specified.
And now while troubleshooting the cargo door, the hinges on my landing gear no longer operate properly.
I've attached the ship and spawning it in from blueprint brings the hinge problem with it.

Regarding the blueprint image, it looks like the stage it was at before the last update. Other than that I don't know why the image isn't being updated.
...

### Environment
- Starship EVO version: 21w21b
- Graphics Card: RTX2060

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_
![20210603190857_1](https://user-images.githubusercontent.com/51359034/120693713-44eb9280-c4a1-11eb-9ed5-20f32c13e225.jpg)
![20210603190901_1](https://user-images.githubusercontent.com/51359034/120693719-474dec80-c4a1-11eb-9c33-c046aebfb165.jpg)
![20210603190833_1](https://user-images.githubusercontent.com/51359034/120693737-4ddc6400-c4a1-11eb-8eeb-a114542d3d13.jpg)
![20210603190842_1](https://user-images.githubusercontent.com/51359034/120693746-4fa62780-c4a1-11eb-8541-013d30461573.jpg)

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/6593530/Player.log)
I'm also including the prev log too, as it seems larger than normal
[Player-prev.log](https://github.com/tsunamayo/Starship-EVO/files/6593536/Player-prev.log)

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.
[NOVOX Cargo Ship 01.zip](https://github.com/tsunamayo/Starship-EVO/files/6593562/NOVOX.Cargo.Ship.01.zip)

**Comment by Kaiser-Indrasil** [comment]
Yeah, looks like it's got to do with a collision check because I tried to input a value directly into the Hinge's menu and it didn't move all the way either.

**Comment by CurioInventorium** [comment]
I was finding if I altered the sequence block for which ever value was active, the hinge would move slightly or wobble. At some point after spamming text box loads the hinge suddenly wooshed to the correct position. But then would get stuck going back the other way.
So something is blocking what was originally a freely moving part and there's nothing in the way.

AHHH! I forgot to mention! On this build I deleted quite a few blocks off the wings and I did end up with some strange flickering blocks floating in the area where parts of the right-hand wing used to be. And on the ship itself were a few invisible blocks that when I deleted, the flickering ones disappeared.
So maybe this is some sort of delete overload and its interfering with the collision checks. idk
